cli/app.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 
 	"github.com/urfave/cli/v2"
+	apppb "go.viam.com/api/app/v1"
```

Comment:
Look into why we are adding this dependency now (I would think this would have existed already and adding it is a mistake).

Reply:
Triple check to confirm the dependency on API is okay -- even though inherently this stuff is "dependent" on the API I feel like this is a red herring.

Reply:
Starting to see that this may be okay - it is imported already in this package (only in a different file) - the real question becomes whether we want to hardcode the string values that we import here instead of importing them - to maybe check against breaking changes in our API ... ? Again - deferring to someone who owns this code path